packages/runtime-common/dompurify-runtime.ts

```diff
@@ -1,15 +1,22 @@
 import DOMPurify from 'dompurify';
 
-let domPurify: DOMPurify.DOMPurifyI;
+let _domPurify: DOMPurify.DOMPurifyI | undefined = undefined;
+let customJsdom: any = null;
+
+export function setCustomJsdom(jsdomInstance: any) {
```

Comment:
We need to set the jsdom from the client since this can be used from browser and node js.